autoload/neoformat/formatters/zsh.vim

```diff
@@ -0,0 +1,13 @@
+function! neoformat#formatters#zsh#enabled() abort
+    return ['shfmt']
+endfunction
+
+function! neoformat#formatters#zsh#shfmt() abort
```

Comment:
looks good!

can you also update the readme and docs/neoformat.txt?

Reply:
That was an oversight on my part. Resolved.